src/main/java/org/tndata/android/compass/util/API.java

```diff
@@ -371,6 +373,10 @@ public static String getRandomReward(){
         public static String postResetEmailUrl(){
             return BASE_URL + "reset-password/";
         }
+
+        public static String postLog(){
+            return "https://staging.tndata.org/cronlog/add/";
```

Comment:
I see we're only posting info while in debug, but do we want to prefix this with `BASE_URL` in case we decide to log something in production?